packages/code-du-travail-ui/src/Tabs/index.js

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import { Tab, Tabs as RootTabs, TabList, TabPanel } from "react-tabs";
+import { box, colors, fonts, spacing } from "../theme";
+
+class Tabs extends React.PureComponent {
+  render() {
+    const { data, defaultIndex, onSelect, selectedIndex } = this.props;
+    const tabs = data.map(item => <Tab key={item.key}>{item.tab}</Tab>);
+    const tabContents = data.map(item => (
+      <TabPanel key={item.key}>{item.panel}</TabPanel>
+    ));
+
+    const props = {
+      onSelect,
+      // conditional props
+      ...(typeof selectedIndex === "number" && { selectedIndex }),
+      ...(isNaN(selectedIndex) && { defaultIndex })
+    };
+
+    return (
+      <StyledTabs {...props}>
+        <TabList>{tabs}</TabList>
+        {tabContents}
+      </StyledTabs>
+    );
+  }
+}
+
+Tabs.propTypes = {
+  withHistory: PropTypes.bool,
+  onSelect: PropTypes.func,
+  selectedIndex: PropTypes.number,
+  defaultIndex: PropTypes.number,
+  data: PropTypes.arrayOf(
+    PropTypes.shape({
+      key: PropTypes.string.isRequired,
```

Comment:
Why key is mandatory ? We could infer it from tab name or tab position

Reply:
I want to let the possibility to provide customs keys. But then, if I don't make the key mendatory, I could have key duplication if I fallback on the current index and that such key has already been provided or will be.

Reply:
So I prefer to let the parent component handle all the  key and make them mandatory.

Reply:
Sorry to disagree but I'm not a fan a implementation detail that leak over public compenent api . 
Why do we need to let user define custom key ? We can definitly create some internal ref

Reply:
I agree, let's remove the key then